.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,72 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Doris enhancement
+description: Add an enhanment for Apache Doris
+title: "[Enhancement] "
+labels: [ "kind/enhancement" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for your good enhancement for Doris.
```

Comment:
Please add
`Please help sync to the [Dev Mail List](mailto:dev@doris.apache.org). You will get FASTER help！[How to subscribe](mailto:dev-subscribe@doris.apache.org)`

Reply:
you can refer to `.github/ISSUE_TEMPLATE/feature-request.yml`

Reply:
fixed